tap_recharge/schemas/collections.json

```diff
@@ -15,6 +15,18 @@
     "updated_at": {
       "type": ["null", "string"],
       "format": "date-time"
+    },
```

Comment:
There were only 3 fields `id`, `created_at`, and `updated_at` being replicated from the Tap. So, we added the missing 4 fields in the JSON schema to replicate. [Reference](https://developer.rechargepayments.com/2021-11/collections/collection_object)